src/Sylius/Bundle/ShopBundle/Form/Type/CartItemType.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ShopBundle\Form\Type;
+
+use Sylius\Bundle\OrderBundle\Form\Type\CartItemType as BaseCartItemType;
+use Symfony\Component\Form\AbstractType;
+
+final class CartItemType extends AbstractType
```

Comment:
As [the PR](https://github.com/Sylius/Sylius/pull/16748) has been already merged, this shop CartItemType should be also used in newly added there shop AddToCartType